src/mococrw/openssl_wrap.h

```diff
@@ -679,6 +679,9 @@ void _X509_STORE_add_cert(X509_STORE *store, X509 *cert);
  * Initialize an X509 store context
  *
  * @param ctx The context to be initialized
```

Comment:
Go to line 641 and add bio to the function declaration in order to avoid doxygen warnings. SSL_X509_Ptr _PEM_read_bio_X509(BIO* bio);

Reply:
Done.